pkg/apis/sources/v1alpha1/awsperformanceinsights_types.go

```diff
@@ -48,13 +48,24 @@ var (
 type AWSPerformanceInsightsSourceSpec struct {
 	duckv1.SourceSpec `json:",inline"`
 
+	// ARN of the RDS instance to receive metrics for.
+	// https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonrds.html#amazonrds-resources-for-iam-policies
 	ARN apis.ARN `json:"arn"`
 
-	PollingInterval *apis.Duration `json:"pollingInterval"`
-
-	Credentials AWSSecurityCredentials `json:"credentials"`
+	// Duration which defines how often metrics should be pulled from Amazon Performance Insights.
+	// Expressed as a duration string, which format is documented at https://pkg.go.dev/time#ParseDuration.
+	PollingInterval apis.Duration `json:"pollingInterval"`
 
+	// List of queries that determine what metrics will be sourced from Amazon Performance Insights.
+	//
+	// Each item represents the 'metric' attribute of a MetricQuery.
+	// https://docs.aws.amazon.com/performance-insights/latest/APIReference/API_MetricQuery.html
+	//
+	// +optional
 	MetricQueries []string `json:"metricQueries"`
```

Comment:
are we moving this to optional? do we not need to update the reconcile for that here -> https://github.com/triggermesh/aws-event-sources/blob/fcaa5a70e3cec4e67cfd7fa676be1308ca65777d/pkg/reconciler/awsperformanceinsightssource/adapter.go#L61 @antoineco

Reply:
Oh, right, that wasn't intentional, I did a bad copy-paste from the CloudWatch source!

Reply:
I was a little confused :)

Reply:
Fixed in [fcaa5a70..0e77c19d](https://github.com/triggermesh/aws-event-sources/compare/fcaa5a70e3cec4e67cfd7fa676be1308ca65777d..0e77c19d78c9e9135c6a2d9667a521fddad2cabd)